config/routes.rb

```diff
@@ -212,6 +212,11 @@
     end
   end
 
+  namespace :super_admin do
```

Comment:
IMO Superadmin namespace is misleading at the moment. We are not creating a super admin, we just create an account.